snuba/query/query.py

```diff
@@ -11,6 +11,7 @@
     TypeVar,
     Union,
 )
+from dataclasses import dataclass
```

Comment:
please move it above, we try to keep import alphabetically sorted .... according to an alien alphabet that is the same as the latin one 70% of the times.